doxygen/contributor_help_pages/getting_started.md

```diff
@@ -283,16 +314,16 @@ the arena and then the objects saved here will be available during the reverse
 pass.
 
 There is a utility for saving objects into the arena that do need their
-destructors called. This is discussed under `make_callback_ptr()`. Arena types
+destructors called. This is discussed under @ref stan::math::make_chainable_ptr . Arena types
 are helper functions and types for easily saving variables to the arena.
 
-`arena_t<T>` defines, for a given type `T`, the type of a lightweight
+@ref stan::arena_t defines, for a given type `T`, the type of a lightweight
 object pointing to memory in the arena that can store a `T`. Memory for
-`arena_t<T>` objects is only recovered when `recover_memory` is called.
-No destructors are called for objects stored with `arena_t<T>()`.
+@ref stan::arena_t objects is only recovered when @ref stan::math::recover_memory is called.
+No destructors are called for objects stored with @ref stan::arena_t.
 
-Copying a variable `x` to the arena can be done with either `to_arena()` or
-`arena_t<T>`:
+Copying a variable `x` to the arena can be done with either @ref stan::math::to_arena or
```

Comment:
`to_arena` is actually a wrapper that just calls the appropriate `arena_t<>` type trait for you. I rewrote this to explain that users should use `to_arena` instead of directly using `arena_t`